### PR TITLE
Create AddTodoForm and TodoList components

### PR DESCRIPTION
**Pull request by EvgeniiRychkov**

### CODE REVIEW

File: src/AddTodoForm.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function AddTodoForm() {
+  return (
+    <form>
+      <label htmlFor="todoTitle">Title </label>
+      <input id="todoTitle" type="text" />
+      <button>Add</button>
```
Comment (AnnaSolovykh) [inline src/AddTodoForm.jsx]: Adding type="submit" to the button is a good practice to define its behavior in the form.

**Review by AnnaSolovykh — APPROVED** [review]
Great job, Evgenii! Looks good to me!

File: src/AddTodoForm.jsx (hunk 1 above)
Comment (EvgeniiRychkov) [reply src/AddTodoForm.jsx]: Hi Anna, I changed it

Comment (AnnaSolovykh) [reply src/AddTodoForm.jsx]: Well done!

**Pull request merged**